docs/book_examples/src/env_md.rs

```diff
@@ -2,7 +2,8 @@ use druid::widget::Label;
 use druid::{Color, Key, WidgetExt};
 
 // ANCHOR: key_or_value
-const IMPORTANT_LABEL_COLOR: Key<Color> = Key::new("my-app.important-label-color");
+const IMPORTANT_LABEL_COLOR: Key<Color> =
+    Key::new("org.linebender.druid.theme.my-app.important-label-color");
```

Comment:
Since this is in documentation rather than druid itself, I'd go for something like "org.my-app.important-label-color"